doc/sphinx/source/recipes/recipe_iht_toa.rst

```diff
@@ -0,0 +1,123 @@
+
+Recipes are stored in esmvaltool/recipes/
+
+    * recipe_iht_toa.yml calculates the IHT maps for the following radiative fluxes:
+
+        * Total net, SW net, LW net (Figure 2).
+        * Total CRE, SW CRE, LW CRE (Figure 4).
+        * All-sky and clear-sky reflected SW (Figure 5).
+        * The meridional heat transports (MHT) of the fluxes above (Figures 1 and 3).
+
+Diagnostics are stored in esmvaltool/diag_scripts/iht_toa/
+
+    * single_model_diagnostics.py: driver script that produces the plots.
+    * poisson_solver.py: solver that calculates the IHTs.
+
+.. _`Pearce and Bodas-Salcedo (2023)`:  https://doi.org/10.1175/JCLI-D-22-0149.1
+
+User settings in recipe
+-----------------------
+
+
+Variables
+---------
+
+* rlut (atmos, monthly, longitude latitude time)
+* rlutcs (atmos, monthly, longitude latitude time)
+* rsutcs (atmos, monthly, longitude latitude time)
+* rsut (atmos, monthly, longitude latitude time)
+* rsdt (atmos, monthly, longitude latitude time)
+
+Observations and reformat scripts
+---------------------------------
+
+* CERES-EBAF
+
+References
+----------
+
+* Pearce, F. A., and A. Bodas-Salcedo, 2023: Implied Heat Transport from CERES
+  Data: Direct Radiative Effect of Clouds on Regional Patterns and Hemispheric
+  Symmetry. J. Climate, 36, 4019–4030, doi: 10.1175/JCLI-D-22-0149.1.
+
+Example plots
+-------------
+
+.. _fig_iht_toa_1:
+.. figure::  /recipes/figures/iht_toa/figure1_CERES-EBAF_CERES-EBAF.png
+   :align:   center
+
+   The implied heat transport due to the total net flux (blue), split into
+   the contributions from the SW (orange) and LW (green).
+
+.. _fig_iht_toa_2:
+.. figure::  /recipes/figures/iht_toa/figure2_CERES-EBAF_CERES-EBAF.png
+   :align:   center
+
+   The energy flux potentials for (a) TOT, (c) SW, and (e) LW fluxes,
+   alongside maps of the spatial anomalies of the fluxes [(b),(d),(f)
+   flux minus global average flux, respectively].
+   The implied heat transport is calculated as the gradient of the energy
+   flux potential, shown by the white vector arrows for a subset of points
+   to give the overall transport pattern.
+   Heat is directed from the blue minima of the potential field to
+   yellow maxima, with the magnitude implied by the density of contours.
+   All maps of the same type share the same color bar at the bottom
+   of the column.
+
+.. _fig_iht_toa_3:
+.. figure::  /recipes/figures/iht_toa/figure3_CERES-EBAF_CERES-EBAF.png
+   :align:   center
+
+   Direct radiative effects of clouds on the meridional heat transport.
+   (a) Contributions from TOT CRE (blue), SW CRE (orange), and LW CRE (green)
+   fluxes. (b) Contributions from all-sky and clear-sky OSR.
+   In (b), both curves have been multiplied by −1 such that positive heat
+   transport is northward.
+
+.. _fig_iht_toa_4:
+.. figure::  /recipes/figures/iht_toa/figure4_CERES-EBAF_CERES-EBAF.png
+   :align:   center
+
+   As in Figure 2, but for the implied heat transport associated with
```

Comment:
Note that figure numbers appear to be dynamic (https://esmvaltool--3177.org.readthedocs.build/en/3177/recipes/recipe_iht_toa.html#example-plots) so Fig 2 actually showing as Fig 39 in this build. Hopefully possible to do a dynamic reference?

Reply:
Well spotted @alistairsellar . I don't know how to include a dynamic reference. Happy to do it if someone points me to the right documentation. I think it would be even better if the numbering could be reset to start from 1 for each recipe, that way each recipe would be more self-contained.

Reply:
I've not done this for figures or numbered elements before, but if I understand https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-numref-role then I think ```:numref:`fig_iht_toa_2` ``` might work. If not, I guess another option is to replace "As in Figure 2" with a copy/merge of the Fig 2 caption so that the caption is self-contained.